resources/qml/ActionButton.qml

```diff
@@ -64,6 +70,20 @@ Button
             horizontalAlignment: Text.AlignHCenter
             elide: Text.ElideRight
         }
+
+        //Icon if displayed on the right side.
+        UM.RecolorImage
+        {
+            id: buttonIconRight
+            source: buttonIconLeft.source
+            height: buttonText.height
+            width: visible ? height : 0
+            sourceSize.width: width
```

Comment:
Maybe we should put this as a default in RecolorImage since we almost always do this.

Reply:
Done: Ultimaker/Uranium#8cb2740150048e4e39fdc74a3a56b1b52845974b and 4d87c464237c4c92c2aa5dfd0c4909a94f49bf92